### ISSUE TITLE
Function set_token_snake_metadata_entry Lacks Proper Parameter Definition

### ISSUE THREAD

**Issue by DanGlChris** [ton-blockchain/stablecoin-contract#63]
Function set_token_snake_metadata_entry does not properly define its parameter value, which may result in an error when the function is called. 

![image](https://github.com/ton-blockchain/stablecoin-contract/assets/49105704/51080ac6-18b0-495d-bcd8-916540535370)

**Comment by nns2009** [comment]
In FunC:
`t = a~fun(b, c)`
is the same as
`a, t = fun(a, b, c)`
so the first parameter is "content_dict"

**Comment by tolya-yanot** [comment]
Yes, it's a feature of the syntax. @nns2009 answered you correctly
